src/watchkit.cs

```diff
@@ -648,6 +648,14 @@ interface WKInterfaceTable {
 		[Watch (3,0)][NoiOS]
 		[Export ("performSegueForRow:")]
 		void PerformSegue (nint row);
+
+		[Watch (5,1), NoiOS]
```

Comment:
@dalexsoto `xtro` is based on sharpie/headers so it will always match
it does not mean it's in 5.1 - you have to run intro on a 5.1 simulator (even better device) to be sure it was present (and not something postponed)

Reply:
I am sorry I meant intro in the description